
std_msgs/Header header

uint32 height
uint32 width

sensor_msgs/PointField[] fields

bool is_bigendian

uint32 point_step

uint32 row_step

bool is_dense